contracts/deploy/mainnet/107_arm_buyback.js.js

```diff
@@ -0,0 +1,61 @@
+const addresses = require("../../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "107_arm_buyback",
+    // forceSkip: true,
+    // onlyOnFork: true, // this is only executed in forked environment
+    deployerIsProposer: false, // just to solve the issue of later active proposals failing
+    proposalId: "",
+  },
+  async ({ deployWithConfirmation, getTxOpts, withConfirmation }) => {
+    const cSwapper = await ethers.getContract("Swapper1InchV5");
+
+    // Deployer Actions
+    // ----------------
+    const { deployerAddr } = await getNamedAccounts();
+    const sDeployer = await ethers.provider.getSigner(deployerAddr);
+
+    // 1. Deploy new ARM Buyback proxy
+    const dARMBuybackProxy = await deployWithConfirmation("ARMBuybackProxy");
+    const cARMBuybackProxy = await ethers.getContract("ARMBuybackProxy");
+
+    // 2. Deploy new ARM Buyback implementation
+    const dARMBuybackImpl = await deployWithConfirmation("ARMBuyback", [
+      addresses.mainnet.WETH,
+      addresses.mainnet.OGN,
+      addresses.mainnet.CVX,
+      addresses.mainnet.CVXLocker,
+    ]);
+    const cARMBuyback = await ethers.getContractAt(
+      "ARMBuyback",
+      dARMBuybackProxy.address
+    );
+
+    // 3. Construct initialize call data to initialize and configure the new buyback
+    const initData = cARMBuyback.interface.encodeFunctionData(
+      "initialize(address,address,address,address,uint256)",
+      [
+        cSwapper.address, // Swapper1InchV5
+        addresses.mainnet.MultiSig2OutOf9, // MS 2
```

Comment:
you can get the strategist address with
```js
const { strategistAddr } = await getNamedAccounts();
```

This is configured in the Hardhat config